packages/text-input/src/TextInput/TextInput.tsx

```diff
@@ -51,7 +52,8 @@ const TextInput = React.forwardRef<HTMLInputElement, TextInputProps>(
       onChange,
       onBlur,
       placeholder,
-      errorMessage,
+      errorMessage = 'This input needs your attention',
```

Comment:
Where did this default message come from?

Reply:
These came from design as part of the new design specs for all inputs. Rob specified this copy for the defaults

Reply:
@bruugey should we be setting a default error message?

Reply:
I believe the specs said to this time - correct me if I'm wrong @audroue